cmake/public/cuda.cmake

```diff
@@ -347,12 +347,7 @@ list(APPEND CUDA_NVCC_FLAGS ${NVCC_FLAGS_EXTRA})
 message(STATUS "Added CUDA NVCC flags for: ${NVCC_FLAGS_EXTRA}")
 
 # disable some nvcc diagnostic that appears in boost, glog, glags, opencv, etc.
-foreach(diag cc_clobber_ignored
-             set_but_not_used field_without_dll_interface
-             base_class_has_different_dll_interface
-             dll_interface_conflict_none_assumed
-             dll_interface_conflict_dllexport_assumed
-             bad_friend_decl)
+foreach(diag cc_clobber_ignored)
```

Comment:
Have you checked the build logs if we run into any of these?

Reply:
No noticeable number of increased warnings. But I will re-check it locally.